src/Microsoft.Extensions.Http/Logging/EventIds.cs

```diff
@@ -8,20 +8,15 @@ namespace Microsoft.Extensions.Http.Logging
     internal static class EventIds
```

Comment:
This is internal and all event ids are nested now. Suggest removing this entirely.

Reply:
Oh oops, this is supposed to be gone